utils/utils.go

```diff
@@ -263,6 +264,15 @@ func CreateClient(conf NetConf) (*client.Client, error) {
 	return calicoClient, nil
 }
 
+func IsReady(client *client.Client) bool {
```

Comment:
Can you add a comment explaining the purpose of this, so people don't wonder what this is for in future